kong/kong_test.go

```diff
@@ -98,6 +98,14 @@ func runWhenKong(t *testing.T, semverRange string) {
 		if rcIndex != -1 {
 			v = v[:rcIndex]
 		}
+		eeIndex := strings.Index(v, "-enterprise-edition")
+		if eeIndex != -1 {
+			v = v[:eeIndex]
+		}
+		// A bit of a hack. Kong sometimes reports a version that
+		// does not match semver conventions. i.e. "1.5.0.1" will
+		// not parse properly.
+		v = strings.Join(strings.Split(v, ".")[:3], ".")
```

Comment:
I'd recommend to implement this logic into a new method that can be re-used if need be. The previous logic was simple enough but with inclusion of this and the "rc" hack above that, we should probably refactor it. If you feel that's a scope creep to this PR and should rather be done separately, that's fine too.

Reply:
I think you're right, but I'm going to defer this refactor to a separate PR